integration/kubernetes/k8s-ro-volume.bats

```diff
@@ -36,12 +36,19 @@ setup() {
 	sudo ls -lR $host_mounts/ | grep $tmp_file && echo "should not find $tmp_file in host mounts" && false
```

Comment:
seems like `shared_mounts` and `host_mounts` are not valid in runtime-rs, is the check here also needed to be changed?

Reply:
Yes, I will fix that